src/backend/catalog/system_views.sql

```diff
@@ -26,8 +26,14 @@ CREATE VIEW pg_roles AS
         rolcanlogin,
         rolreplication,
         rolconnlimit,
+        rolenableprofile,
+        pg_profile.prfname AS rolprofile,
+        rolaccountstatus,
+        rolfailedlogins,
         '********'::text as rolpassword,
         rolvaliduntil,
+        rollockdate,
```

Comment:
This view is defined by Postgres, we should not change it. If we want to add a view to show fields added by this pr, add a new one with a name like pg_roles_with_profile which only have necessary columns. 
And this lacks of a test case of this view.

Reply:
Like other catalog tables modifications, it's implemented same as EDB.

Reply:
I will add a test case in regress.